src/utils/datoUtils.ts

```diff
@@ -309,3 +309,11 @@ export const isExpiredForhandsvarsel = (
 
   return false;
 };
+
+export const getForhandsvarselFrist = (): Date => {
```

Comment:
Synes vi burde ha en `forhandsvarselUtils` e.l. hvor denne burde ligge siden dette er domenelogikk og ikke generell dato-utils som det andre her.

Reply:
Et alternativ er også å bare legge det rett i `SendForhandsvarselSkjema.tsx`. Det er vel kanskje det eneste stedet det skal brukes? 😊

Reply:
Ja, det går også an. Hvis det blir samme logikk for 8-4 forhåndsvarsel så er det fint å samle det i en util/hook utenfor komponentene.